modules/sugar/src/main/ts/ephox/sugar/impl/ClickBehavior.ts

```
@@ -0,0 +1,23 @@
+const preventDefaultClickLinkBehavior = (isMetaKeyPressed: (e: MouseEvent) => boolean): void => {
+  document.addEventListener('click', (e) => {
+    for (let elm = e.target as Node | null; elm; elm = elm.parentNode) {
+      if (elm.nodeName === 'A') {
+        const href = (elm as HTMLAnchorElement).getAttribute('href');
+
+        if (href && href.startsWith('#')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)